packages/flutter_tools/lib/src/features.dart

```diff
@@ -53,7 +53,7 @@ abstract class FeatureFlags {
   /// Whether a particular feature is enabled for the current channel.
   ///
   /// Prefer using one of the specific getters above instead of this API.
-  bool isEnabled(Feature feature) => false;
+  bool isEnabled(Feature feature);
```

Comment:
this default was never called; remove the implementation, so that future invocations will force sub-classes to implement.